sunbeam/commands/bootstrap.py

```diff
@@ -62,20 +67,27 @@
 @click.command()
 @click.option(
     "--role",
```

Comment:
`Role` now represents a repeatable option that aggregates into a list. Makes more sense to be plural.
```suggestion
    "--roles",
```